pkgs/tools/security/trueseeing/default.nix

```diff
@@ -15,22 +15,20 @@ python3.pkgs.buildPythonApplication rec {
     hash = "sha256-g5OqdnPtGGV4wBwPRAjH3lweguwlfVcgpNLlq54OHKA=";
   };
 
-  postPatch = ''
-    substituteInPlace pyproject.toml \
-      --replace "attrs~=21.4" "attrs>=21.4"
-  '';
-
   nativeBuildInputs = with python3.pkgs; [
     flit-core
+    pythonRelaxDepsHook
   ];
 
+  pythonRelaxDeps = true;
+
   propagatedBuildInputs = with python3.pkgs; [
     attrs
-    ipython
     jinja2
     lxml
     pypubsub
     pyyaml
+    termcolor
```

Comment:
I don't understand why `pythonRelaxDeps` is used since `termcolor` is added, and the failure of the build initially was due to missing `termcolor`.

Reply:
Their pyproject.toml specifies an outdated version of termcolor.

Reply:
So even by adding termcolor as a dependency, it fails to build thus is `pythonRelaxDeps` necessary ?

Reply:
We should probably open an issue upstream to let them know that they should bump the termcolor dependency, could you take care of it ?

Reply:
@dotlambda Bump was merged upstream, can you please upgrade your PR ? to remove the pythonRelaxDeps ?

Reply:
There hasn't been a new release and it would break again once termcolor is updated to 2.5.

Reply:
I will ask for a new release then, if that's possible, if not then, we'll leave it as that.

Reply:
I will merge this anyway because, as I said, it would break again in the future.

Reply:
Here is the new release : https://github.com/alterakey/trueseeing/pull/358#issuecomment-1900368565 @dotlambda